opac/webapp/main/views.py

```diff
@@ -175,7 +175,7 @@ def article_html_by_aid(article_id):
     if not article.htmls:
         abort(404, _(u'HTML do artigo não encontrado'))
 
-    article_html = article.htmls[0].source
+    article_html = article.htmls[0].url
```

Comment:
Se entendi....

Com certeza vai quebra a interface, mas o que sugere que seja feito nesse
PR o objetivo é rodar os testes para o modelo Article a página do Artigo
será ajustada em outra tarefa, mantive um param de retorno somente,
inclusive precisa do contexto do idioma para que seja considerado
totalmente correto.

Sugiro deixarmos assim e na tarefa de mostrar o artigo resolvermos

On Thu, Apr 7, 2016 at 2:23 PM, Juan Funez <notifications@github.com> wrote:

> In opac/webapp/main/views.py
> <https://github.com/scieloorg/opac/pull/163#discussion_r58910506>:
>
> > @@ -175,7 +175,7 @@ def article_html_by_aid(article_id):
> >      if not article.htmls:
> >          abort(404, _(u'HTML do artigo não encontrado'))
> >
> > -    article_html = article.htmls[0].source
> > +    article_html = article.htmls[0].url
>
> tenta acessar essa view no browser, aonde esperava um string com o html do
> artigo que seria jogada dentro de um <iframe ../> agora vem uma url.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/scieloorg/opac/pull/163/files/ea2456c34a2e0fc38beb64817a32ea5a586dc3fd#r58910506>
>



-- 

Jamil Atta Junior